app/experimenter/nimbus-ui/.storybook/design-docs/DocsFormValidation/index.stories.tsx

```diff
@@ -0,0 +1,186 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useState, useCallback } from "react";
+import { storiesOf } from "@storybook/react";
+import { ProtoForm, TestCases } from "./helpers";
+import { Container, Alert } from "react-bootstrap";
+import LinkExternal from "../../../src/components/LinkExternal";
+import InlineErrorIcon from "../../../src/components/InlineErrorIcon";
+
+const MODELS_URL =
+  "https://github.com/mozilla/experimenter/blob/main/app/experimenter/experiments/models/nimbus.py";
+
+storiesOf("Design Docs/Form Validation", module)
+  .add("client-side validation", () => {
+    const [submitErrors, setSubmitErrors] = useState<Record<string, any>>({});
+
+    return (
+      <Container className="pt-5">
+        <h2>Client-side Validation</h2>
+        <p>
+          Client-side validation should be used to prevent invalid mutations
+          from being attempted (i.e. the wrong type or fields that cannot be{" "}
+          <code>null</code> in the{" "}
+          <LinkExternal href={MODELS_URL}>database</LinkExternal>
+          ). It should be used to validate fields that are required for launch
+          or to duplicate server-side validation.
+        </p>
+
+        <ProtoForm
+          demoInputs={[
+            { name: "name", label: "Branch Name", defaultValue: "" },
+          ]}
+          isLoading={false}
+          isServerValid={true}
```

Comment:
Since this will implicitly be `true` without `={true}` this line could just be `isServerValid` for some syntactic sugar, but totally fine to leave as-is